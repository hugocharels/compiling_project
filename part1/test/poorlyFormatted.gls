(
        IF                              abbb

        THEN
            x=    3
        :

        abbbTHENx = 5

        IFabbbTHENx=3
