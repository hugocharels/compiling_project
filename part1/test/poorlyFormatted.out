token: (                lexical unit: LPAREN
token: IF               lexical unit: IF
token: abbb             lexical unit: VARNAME
token: THEN             lexical unit: THEN
token: x                lexical unit: VARNAME
token: =                lexical unit: ASSIGN
token: 3                lexical unit: NUMBER
token: :                lexical unit: COLUMN
token: abbbTHENx        lexical unit: VARNAME
token: =                lexical unit: ASSIGN
token: 5                lexical unit: NUMBER
token: IFabbbTHENx      lexical unit: PROGNAME
token: =                lexical unit: ASSIGN
token: 3                lexical unit: NUMBER

Variables
abbb    2
abbbTHENx       8
x       5
